@kiva/kv-components/vue/KvWwwHeader/LendMenu/KvLendMenu.vue

```diff
@@ -61,797 +61,47 @@ export default {
 		});
 
 		const onLoad = async (apollo) => {
-			// apollo.watchQuery({
-			// 	query: gql`query countryFacets {
-			// 		lend {
-			// 			countryFacets {
-			// 				count
-			// 				country {
-			// 					name
-			// 					region
-			// 					isoCode
-			// 				}
-			// 			}
-			// 		}
-
-			// 	}`,
-			// }).subscribe({
-			// 	next: ({ data }) => {
-			// 		countryFacets.value = data?.data?.lend?.countryFacets ?? [];
-			// 		isRegionsLoading.value = false;
-			// 	},
-			// });
+			apollo.watchQuery({
```

Comment:
Do we need a loading state while this data is fetched? I know we had one in the old lend menu, so maybe it was already ported here, I just didn't see the changes in this PR.